TripHouse/main.js

```diff
@@ -51,17 +51,60 @@ const data = [
     },
   ];
 
-const homesContent = document.querySelector('.homes_content')
-
+const homesContent = document.querySelector('.homes_content');
 function createCard1(data) {
   return `
   <div class="content_img col-xs-6 col-sm-6">
     <img src="${data.imageUrl}" alt="${data.name}">
     <p class="accent-blue">${data.name}</p>
     <p class="secondary">${data.city}, ${data.country}</p>
   </div>`
+};
+data.forEach((element, index) => {
+  if (index < 4) {
```

Comment:
It's better to move values like 4 into variables - cause it looks like an option that can be changed in the future